packages/firebase_storage/lib/firebase_storage.dart

```diff
@@ -48,6 +48,13 @@ class StorageReference {
     );
   }
 
+  Future<String> getDownloadURL() async {
+    final dynamic downloadUrl = await FirebaseStorage._channel.invokeMethod(
+        "StorageReference#getDownloadUrl",
+        <String, String>{'path': _pathComponents.join("/")});
```

Comment:
If you put a trailing comma here you can avoid the double-indent

Reply:
Done